packages/calypso-e2e/src/lib/components/support-component.ts

```diff
@@ -179,8 +182,8 @@ export class SupportComponent {
 	 * @returns {Promise<Page>} Reference to support page.
 	 */
 	async visitArticle(): Promise< Page > {
-		// Wait for the placeholder to disappear from the Support article preview.
-		await this.page.waitForSelector( selectors.supportArticlePlaceholder, { state: 'hidden' } );
+		const visitArticleHandle = await this.page.waitForSelector( selectors.visitArticleButton );
+		await visitArticleHandle.waitForElementState( 'stable' );
```

Comment:
It may be worth trying to use the `Locator` API here:
https://playwright.dev/docs/1.14/api/class-locator#locator-click

Reply:
Oh, I love that idea! I had no idea That API existed, but we've had lots of issues where `elementHandles` get detached - I really like the `Locator` API as an alternative! not required though of course :)

Reply:
Same, it was only by chance that I came across this API. I will keep that in mind and if issues crop up again, I will try switching to the Locator API.